recipe/meta.yaml

```diff
@@ -21,8 +21,7 @@ requirements:
     - pkg-config
 
   host:
-    - grpc-cpp
-    - fmt
+    - grpc-cpp <=1.42.0
```

Comment:
We shouldn't put explicit pins in host. if you need an older build, this should go in a local conda_build_config.yaml in the recipe.

Reply:
Also, if this results in a downgrade of the grpc-cpp pinning, then we need to merge to a branch.